components/docs-chef-io/content/automate/backup.md

```diff
@@ -1,13 +1,13 @@
 +++
-title = "Backup & Restore"
+title = "Backup"
 
 date = 2018-03-26T15:27:52-07:00
 draft = false
 [menu]
   [menu.automate]
-    title = "Backup & Restore"
+    title = "Backup"
     parent = "automate/getting_started"
-    identifier = "automate/getting_started/backup.md Backup & Restore"
+    identifier = "automate/getting_started/backup.md Backup"
```

Comment:
Before this change, the table of contents showed a page for "Backup & Restore" and a page for "Restore"

Reply:
Thank you for catching this. I'm investigating this further.